charts/bc-apis/README.md

```diff
@@ -0,0 +1,50 @@
+# BC APIS Helm Chart
+
+This Helm chart installs the `bc-apis` into your Kubernetes (v1.16+) or Openshift cluster (v4.3+).
+
+## In Helm
+
+### Install bc-apis
+
+> NOTE: Use `bc-apis` as release name
+
+```shell
+    helm install bc-apis kubebb/bc-apis
+```
+
+### Upgrade bc-apis
+
+```shell
+    helm upgrade bc-apis kubebb/bc-apis
+```
+
+### Uninstall bc-apis
+
+```shell
+    helm delete bc-apis
+```
+
+## Configuration
+
+| Parameter                    | Description | Default                                       |
+|------------------------------|-------------|-----------------------------------------------|
+| `ingressDomain`              | B           |                                               |
```

Comment:
Description needs to be supplemented.

Reply:
maybe `Domain of the ingress controller that exposes the application to the outside world`